.circleci/config.yml

```diff
@@ -46,14 +49,17 @@ jobs:
           command: make build
           name: Build image on the cluster
       - run:
-          command: make install_dev
+          command: >
+            make install_dev
+            MASTER_CPU_REQUEST="300m" MASTER_CPU_LIMIT="300m"
+            WORKER_CPU_REQUEST="300m" WORKER_CPU_LIMIT="300m"
+            MASTER_MEMORY_REQUEST="512Mi" MASTER_MEMORY_LIMIT="512Mi"
+            WORKER_MEMORY_REQUEST="512Mi" WORKER_MEMORY_LIMIT="512Mi"
+            POSTGRESQL_WORKERS=2
```

Comment:
maybe we should read env var to check if currently on CI and put this logic in makefile...